lms/templates/courseware/info.html

```diff
@@ -69,7 +69,7 @@ <h2 class="hd hd-2 page-title">${_("Welcome to {org}'s {course_name}!").format(o
               <div class="page-subtitle">${course.display_name_with_default}</div>
             </h2>
         </div>
-        % if last_accessed_courseware_url:
+        % if last_accessed_courseware_url and user_is_enrolled:
```

Comment:
Nit: You could rename `last_accessed_courseware_url` to `resume_course_url` and make it None if user is not enrolled, and not pass in `user_is_enrolled`.

This is on a page that is hopefully going away, so this change is not required.